src/multichoice/style.ts

```diff
@@ -58,7 +58,8 @@ export const MULTICHOICE_CSS = `
     padding: 6px 0;
     overflow-y: auto;
     overflow-x: hidden;
-    max-height: 134px;
+    height: var(--kuc-multi-choice-menu-height, auto);
+    max-height: var(--kuc-multi-choice-menu-height, 134px);
     width: 100%;
```

Comment:
Please add custom-width for width:
width: var(--kuc-multi-choice-menu-width, auto);